README.md

```diff
@@ -114,46 +101,94 @@ can be linked as an external tool in CMSSW:
+     ```
+   - Find the line that starts with process.mix.input.fileNames and change it to (supposing that the PU files are available locally, as is the case at the UCSD machines):
+     ```python
+     process.mix.input.fileNames = cms.untracked.vstring(['file:/data2/segmentlinking/PUSamplesForCMSSW1263/CMSSW_12_3_0_pre5/RelValMinBias_14TeV/GEN-SIM/123X_mcRun4_realistic_v4_2026D88noPU-v1/066fc95d-1cef-4469-9e08-3913973cd4ce.root', 'file:/data2/segmentlinking/PUSamplesForCMSSW1263/CMSSW_12_3_0_pre5/RelValMinBias_14TeV/GEN-SIM/123X_mcRun4_realistic_v4_2026D88noPU-v1/07928a25-231b-450d-9d17-e20e751323a1.root', 'file:/data2/segmentlinking/PUSamplesForCMSSW1263/CMSSW_12_3_0_pre5/RelValMinBias_14TeV/GEN-SIM/123X_mcRun4_realistic_v4_2026D88noPU-v1/26bd8fb0-575e-4201-b657-94cdcb633045.root', 'file:/data2/segmentlinking/PUSamplesForCMSSW1263/CMSSW_12_3_0_pre5/RelValMinBias_14TeV/GEN-SIM/123X_mcRun4_realistic_v4_2026D88noPU-v1/4206a9c5-44c2-45a5-aab2-1a8a6043a08a.root', 'file:/data2/segmentlinking/PUSamplesForCMSSW1263/CMSSW_12_3_0_pre5/RelValMinBias_14TeV/GEN-SIM/123X_mcRun4_realistic_v4_2026D88noPU-v1/55a372bf-a234-4111-8ce0-ead6157a1810.root', 'file:/data2/segmentlinking/PUSamplesForCMSSW1263/CMSSW_12_3_0_pre5/RelValMinBias_14TeV/GEN-SIM/123X_mcRun4_realistic_v4_2026D88noPU-v1/59ad346c-f405-4288-96d7-795f81c43fe8.root', 'file:/data2/segmentlinking/PUSamplesForCMSSW1263/CMSSW_12_3_0_pre5/RelValMinBias_14TeV/GEN-SIM/123X_mcRun4_realistic_v4_2026D88noPU-v1/7280f5ec-b71d-4579-a730-7ce2de0ff906.root', 'file:/data2/segmentlinking/PUSamplesForCMSSW1263/CMSSW_12_3_0_pre5/RelValMinBias_14TeV/GEN-SIM/123X_mcRun4_realistic_v4_2026D88noPU-v1/b93adc85-715f-477a-afc9-65f3241933ee.root', 'file:/data2/segmentlinking/PUSamplesForCMSSW1263/CMSSW_12_3_0_pre5/RelValMinBias_14TeV/GEN-SIM/123X_mcRun4_realistic_v4_2026D88noPU-v1/c7a0aa46-f55c-4b01-977f-34a397b71fba.root', 'file:/data2/segmentlinking/PUSamplesForCMSSW1263/CMSSW_12_3_0_pre5/RelValMinBias_14TeV/GEN-SIM/123X_mcRun4_realistic_v4_2026D88noPU-v1/e77fa467-97cb-4943-884f-6965b4eb0390.root'])
+     ```
+   - Modify the input and output file names accordingly, as well as the number of events.
+
+Then, run the configuration file with `cmsRun`.
+
+To get the DQM files, one would have to run step4 of the 21034.1 workflow with the following modifications:
+   - Modify the `--pileup_input` flag as above, add `--no_exec` to the end of command and then run it.
+   - Modify the output configuration file by including the appropriate PU files, as above, and by changing the input file (the one containing `inDQM` from the previous step) and number of events accordingly.
+
+Running the configuration file with `cmsRun`, the output file will have a name starting with `DQM`. The name is the same every time this step runs,
+so it is good practice to rename the file, e.g. to `tracking_Iters01LST.root`.
+The MTV plots can be produced with the command:
+```bash
+makeTrackValidationPlots.py --extended tracking_Iters01LST.root
```

Comment:
Hi Manos, I ran the step 4 but it says

%MSG-i CUDAService:  (NoModuleName) 02-Jun-2023 08:38:18 PDT pre-events
CUDA runtime version 11.5, driver version 11.7, NVIDIA driver version 515.65.01
CUDA device 0: NVIDIA A30 (sm_80)
%MSG
02-Jun-2023 08:38:22 PDTSuccessfully opened file file:step3_RAW2DIGI_RECO_VALIDATION_DQM_PU_inDQM.root
%MSG-e DQMScaleToClient:  DQMScaleToClient:KshortMonitorClient@endProcessBlock  02-Jun-2023 08:38:22 PDT post-events
MEs not found! Tracking/V0Monitoring/Ks/v0_Lxy not found 
%MSG
%MSG-e DQMScaleToClient:   DQMScaleToClient:LambdaMonitoringClient@endProcessBlock  02-Jun-2023 08:38:22 PDT post-events
MEs not found! Tracking/V0Monitoring/Lambda/v0_Lxy not found 
%MSG
%MSG-w TrackFoldedOccupancyClient:   TrackFoldedOccupancyClient:TrackerMapFoldedClient@endProcessBlock  02-Jun-2023 08:38:22 PDT post-events
MEs needed for this module not found. Skipping.
%MSG
%MSG-w TrackFoldedOccupancyClient:   TrackFoldedOccupancyClient:TrackerMapFoldedClient_highpurity_dzPV0p1@endProcessBlock  02-Jun-2023 08:38:22 PDT post-events
MEs needed for this module not found. Skipping.
%MSG
%MSG-w TrackFoldedOccupancyClient:   TrackFoldedOccupancyClient:TrackerMapFoldedClient_highpurity_pt0to1@endProcessBlock  02-Jun-2023 08:38:22 PDT post-events
MEs needed for this module not found. Skipping.
%MSG
%MSG-w TrackFoldedOccupancyClient:   TrackFoldedOccupancyClient:TrackerMapFoldedClient_highpurity_pt1@endProcessBlock  02-Jun-2023 08:38:22 PDT post-events
MEs needed for this module not found. Skipping.
%MSG
DQMFileSaver::globalEndRun()
02-Jun-2023 08:38:22 PDTClosed file


I modified the "process.", "process.mix.input.fileNames", and input names accordingly. But in the end the plots are empty

Reply:
This is solved once used the provided "/ceph/cms/store/user/evourlio/LST/step2_21034.1_100Events.root" root file. But wasn't working using the step 2 file I generated.